doc/examples/parallel_planning/parallel_planning_tutorial.rst

```diff
@@ -0,0 +1,162 @@
+How to run multiple planning pipelines in parallel with MoveItCpp?
```

Comment:
This isn't really a correct question
```suggestion
How to run multiple planning pipelines in parallel with MoveItCpp
```
A question would be:
> How do you run multiple planning pipelines in parallel with MoveItCpp?

Since these are titles, they don't need to be full sentences or questions. Removing the question mark should be fine.

Reply:
I'd go even shorter: "Running Multiple Planning Pipelines in Parallel with MoveItCpp"